beacon_node/lighthouse_network/src/discovery/enr.rs

```diff
@@ -67,16 +67,16 @@ impl Eth2Enr for Enr {
             .map_err(|_| "Could not decode the ENR syncnets bitfield")
     }
 
-    fn custody_subnet_count<E: EthSpec>(&self, spec: &ChainSpec) -> Result<u64, &'static str> {
-        let csc = self
-            .get_decodable::<u64>(PEERDAS_CUSTODY_SUBNET_COUNT_ENR_KEY)
-            .ok_or("ENR custody subnet count non-existent")?
-            .map_err(|_| "Could not decode the ENR custody subnet count")?;
+    fn custody_group_count<E: EthSpec>(&self, spec: &ChainSpec) -> Result<u64, &'static str> {
+        let cgc = self
+            .get_decodable::<u64>(PEERDAS_CUSTODY_GROUP_COUNT_ENR_KEY)
+            .ok_or("ENR custody group count non-existent")?
+            .map_err(|_| "Could not decode the ENR custody group count")?;
 
-        if csc >= spec.custody_requirement && csc <= spec.data_column_sidecar_subnet_count {
-            Ok(csc)
+        if cgc >= spec.custody_requirement && cgc <= spec.number_of_custody_groups {
```

Comment:
Can we make this line equal to the one in `compute_peer_custody_groups`? It would be easier to follow.

Reply:
Good eyes! I've change this to the same format: https://github.com/sigp/lighthouse/pull/6736/commits/73cf91a1eb95fcb4b2639ba4b806e74cdc539a63, without the `!`. 

I prefer to avoid negation here for readability, whereas in `compute_peer_custody_groups` it's a bit easier to follow if we return invalid input early - this is my personal preference though, happy to change